lib/templates/webmanifest.js

```diff
@@ -2,12 +2,16 @@ const renderWebManifest = (name, color) => `
 {
 	"name": "Gradebook",
 	"short_name": "Gradebook",
+	"description": "Better grades, together. Gradebook is the only student-built, student-centered Grade Tracking app designed to help you take control of your grades.",
```

Comment:
```suggestion
	"description": "Free grade tracking, built by students. Take control of your grades with Gradebook, the essential student-centered grade tracking app.",
```

Reply:
@joshcosta FYI: https://github.com/gradebook/school-configuration/pull/129#discussion_r990665986

I think we should keep the marketing description in sync with this description. I'm fine changing the marketing description if you want this copy.

Reply:
That's kind of what I was thinking too, since we don't use "Better grades, together" on the marketing site any more. I'm fine if we just want to remove that part

Reply:
So keep the old copy but get rid of that first tagline?

Reply:
I'm fine with that. Can you create a PR to sync the marketing site once we finalize?